config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    'GOOGLE_CLIENT_ID': '799903818203-aq22ofd22c1u1aqa04ts3f43fg2gpr2d.apps.googleusercontent.com',
```

Comment:
takie dane nie powinny znajdować się w publicznym repozytorium. Najlepiej konfigurację jest trzymać w zmiennych środowiskowych bądź w dedykowanym pliku `config` który jest poza repozytorium. Oczywiście na potrzeby zadania to jest wystarczające ale pamiętaj na przyszłość :) 

(https://stackoverflow.com/questions/5869216/how-to-store-node-js-deployment-settings-configuration-files)